app/views/course/assessment/submissions/_guided.html.slim

```diff
@@ -0,0 +1,4 @@
+nav
+  ul.pager
```

Comment:
I checked both, but think `.pager` is actually more suitable, wait I'll give you a screen shot.